setup.py

```diff
@@ -4,13 +4,24 @@
     long_description = fh.read()
 
 INSTALL_REQUIRES = [
-    'endaq-calc>=1.1.0.post2',
-    'endaq-cloud>=1.1.0',
-    'endaq-ide>=1.1.0',
-    'endaq-plot>=1.1.0',
+    "backports.cached-property; python_version<'3.8'",
+    "ebmlite>=3.2.0",
+    "idelib>=3.2.3",
+    "jinja2",
+    "numpy>=1.19.5",
+    "pandas>=1.3",
+    "plotly>=5.3.1",
+    "python-dotenv>=0.18.0",
+    "requests>=2.25.1",
+    "scipy>=1.7.1",
     ]
 
 TEST_REQUIRES = [
+    "hypothesis",
+    "plotly",
```

Comment:
this requirement is already listed on line 13
```suggestion
```

Reply:
`endaq.plot` appears to use it.

Reply:
Previous comment was regarding its removal along with `numpy-quaternion`.

Reply:
> `endaq.plot` appears to use it.

yes that's perfectly fine, but the `setup.py` file and the `requirements.txt` files both list `plotly` as a requirement **TWICE**. what I'm asking is to please remove the second instances from those two files.

Reply:
thanks 👍